### PR TITLE
[BACK-1737] Add async user deletion handler

### PR DESCRIPTION
**Pull request by toddkazakov**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/pkg/errors v0.9.1
-	github.com/tidepool-org/go-common v0.6.2
+	github.com/tidepool-org/go-common v0.6.3-0.20201005135347-39fb0d42f208
```
Comment (derrickburns) [inline go.mod:10]: Should we be depending on an untagged version?

Comment (toddkazakov) [reply go.mod]: Fixed

**Review by derrickburns — DISMISSED** [review]
Lgtm

**Review by derrickburns — APPROVED** [review]
LGTM

**Pull request merged**